src/adhocracy_core/adhocracy_core/authentication/__init__.py

```diff
@@ -90,6 +93,46 @@ def wrapped_view(context, request):
     return wrapped_view
 
 
+def is_header_password(request: IRequest) -> bool:
```

Comment:
I am not sure about the conventions, but I would prefer `has_header_password()` or even `has_password_header()`